frontend/packages/console-app/src/components/quick-starts/loader/QuickStartsTileLoader.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import QuickStartTile from '@console/dev-console/src/components/QuickStartTile';
+import QuickStartsLoader from './QuickStartsLoader';
+
+interface QuickStartsTileLoaderProps {
+  onQuickStartCatalogTileRemove?: () => void;
+}
+
+const QuickStartsTileLoader: React.FC<QuickStartsTileLoaderProps> = (props) => (
```

Comment:
I am not sure if we even need this small utility component.

Reply:
removed this component